examples/c++/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The StableHLO Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_test")
+
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],
+)
+
+cc_binary(
```

Comment:
Wondering if need to build this binary. `cc_test`  is sufficient here, right?

Reply:
Just so you can `bazel run` it ?

Reply:
Yes, or `bazel test`

Reply:
I mean t-- I added cc_binary so you can get a runnable binary.
This will be needed if eventually we create binaries that take any args on input -- so you can give it different arguments to play with.

Reply:
My point was the binary for the  target `stablehlo-add_test` will still be created if we do `bazel test examples/c++:stablehlo-add_test`. 

>> This will be needed if eventually we create binaries that take any args on input 

I see your point. In that case it will be a different target capable, with potentially different source file than what is provide now, with the provision of accepting input args.

Reply:
in this example it doesn't make sense since there is no use of input; but it's setting a template for other examples to follow.